apps/data_handler/handlers/loan_states/nostra_alpha/events.py

```diff
@@ -533,12 +533,15 @@ def process_interest_bearing_collateral_mint_event(self, event: pd.Series) -> No
         # The order of the values in the `data` column is: `user`, `amount`, ``.
         # Example:
         # https://starkscan.co/event/0x07d222d9a70edbe717001ab4305a7a8cfb05116a35da24a9406209dbb07b6d0b_5.
-        user = event["data"][0]
+        data = NostraDataParser.parse_interest_bearing_collateral_mint_event(event["data"])
+        user, amount = data.user, data.amount
```

Comment:
please, be specific which amount, we have `face` and `raw` and here it's very important